src/Symfony/Component/ErrorHandler/Debug.php

```diff
@@ -58,3 +58,17 @@ public static function enable($errorReportingLevel = E_ALL, $displayErrors = tru
         DebugClassLoader::enable();
     }
 }
+
+// to be deprecated in 5.1
```

Comment:
why not trigger a deprecation now already telling people to change their front controller so we can remove it in 5.0?

Reply:
I don't know how that could work since there's no reporting infrastructure at this time. Any idea?

Reply:
See last alternative with deprecation notice

Reply:
So if you use `Symfon\Component\Debug\Debug` class you'll  see a deprecation warning with or without Debug component installed.